test/activation_based/test_distributed_dtensor.py

```
@@ -0,0 +1,879 @@
+        assert isinstance(distributed_model.features[0].neuron, TensorShardMemoryModule)
+        assert distributed_model.features[0].neuron.inner.v.shape[1] == distributed_model.features[0].proj_bn[-2].out_channels
+
+
+@pytest.mark.skipif(
+    not DTENSOR_AVAILABLE,
+    reason="DTensor DeviceMesh APIs are unavailable in the current PyTorch build.",
+)
+def test_high_level_cifar10dvs_vgg_helper():
+    with _single_rank_process_group():
+        model = CIFAR10DVSVGG(dropout=0.0, backend="torch").eval()
+        distributed_model, mesh, analysis = configure_cifar10dvs_vgg_distributed(
```

Ruff (0.15.12)

[warning] Unpacked variable `distributed_model` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)